system-int-test/test/cmr/system_int_test/search/acls/collection.clj

```diff
@@ -295,8 +305,8 @@
           (od/assert-collection-opendata-results-match guest-permitted-collections actual-od))))
 
     (testing "all_revisions"
-      (are2 [collections params]
-        (d/refs-match? collections (search/find-refs :collection params))
+      (util/are3 [collections params]
```

Comment:
are we not referring are3 anymore?

Reply:
Our goal is to use refer rarely so it is always clear where a function comes from. We still use it for things like logging, clojure.test functions, and compojure functions.